src/docs/modules/ROOT/pages/modeling-guidelines.adoc

```diff
@@ -18,70 +18,65 @@ SPDX-License-Identifier: MPL-2.0
 === Notes
 
 * As mentioned in the xref:index.adoc[Introduction], Aspect models are described in RDF Turtle
-  xref:bamm-specification:appendix:appendix.adoc#turtle[[turtle\]]. The aspect specification will
```

Comment:
It seems like most of the changes are (automatic) reformatting. Can you please revert these changes as they make it hard to understand the changes in the content.